include/wx/srchctrl.h

```diff
@@ -16,15 +16,23 @@
 
 #include "wx/textctrl.h"
 
-#if !defined(__WXUNIVERSAL__) && defined(__WXMAC__)
+#if defined(__WXUNIVERSAL__)
+    // nothing, use generic controls
+#elif defined(__WXMAC__)
     // search control was introduced in Mac OS X 10.3 Panther
-    #define wxUSE_NATIVE_SEARCH_CONTROL 1
+    #define wxUSE_NATIVE_SEARCH_CONTROL
```

Comment:
This is an incorrect change, `wxUSE_XXX` symbols should be defined as 0 or 1. `wxHAS_XXX` should be defined or not defined and perhaps it would have been better to use the latter here, but at this stage I'd rather avoid changing it as it doesn't seem important enough.

Reply:
Done.